### PR TITLE
Fix/feedback and bug fixing round

### PR DESCRIPTION
**Pull request by Liberatys**
1. Fix duplicate update
2. Fix update on non created session
3. Add explicit public/private

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,6 +32,14 @@ Administration => Roles & Permissions
     cd assets.src && npm install
     rake provision
```
Comment (schmijos) [inline README.md:33]: This doesn't work like this, right? Isn't this to be executed inside of the running container?

File: assets.src/src/redmine-tracky/controllers/form-controller.ts [hunk 2]
```diff
@@ -9,10 +9,16 @@ export default class extends Controller {
     readonly descriptionTarget!: Element;
     readonly issueTargets!: Array<Element>;
 
+    private connected = false;
```
Comment (schmijos) [inline assets.src/src/redmine-tracky/controllers/form-controller.ts:12]: I don't understand this. Why should targets be connected and the controller not? Shouldn't this work in a hierarchical way?

File: assets.src/src/redmine-tracky/controllers/form-controller.ts [hunk 3]
```diff
@@ -61,24 +71,23 @@ export default class extends Controller {
...
     private dispatchUpdate(form: FormData) {
-        $.ajax({
-            type: "POST",
-            url: window.RedmineTracky.trackerUpdatePath,
-            data: {
-                timer_session: form,
-            },
-            async: true,
-        });
+        if ($('[data-ending-action-buttons]').length) {
```
Comment (schmijos) [inline assets.src/src/redmine-tracky/controllers/form-controller.ts:74]: Why is this not a target or at least a value?

**Review by schmijos — APPROVED** [review]

File: assets.src/src/redmine-tracky/controllers/form-controller.ts (hunk 2 above)
Comment (Liberatys) [reply assets.src/src/redmine-tracky/controllers/form-controller.ts:12]: First all the targets are connected -> after all targets are initially connected the 'controller' is connected.

**Pull request merged**